lib/internal/errors.js

```diff
@@ -85,4 +85,5 @@ module.exports = exports = {
 //
 // Note: Please try to keep these in alphabetical order
 E('ERR_ASSERTION', (msg) => msg);
+E('ERR_UNKNOWN_NATIVE_MODULE', (id) => `No such built-in module: ${id}`);
```

Comment:
Now that the error message is updated to `No such built-in module`, the error code should be changed accordingly?